packages/fxa-auth-server/lib/senders/emails/partials/paymentProvider/en.ftl

```diff
@@ -0,0 +1,6 @@
+payment-method = Payment method: PayPal
+credit-card = Credit card:
+# Variables:
+#  $cardType (String) - The name of the credit card, e.g. Visa
```

Comment:
I think we could maybe say, "The name of the credit card payment network, e.g. Visa or Mastercard", or say "The type of credit card" instead of "name" just to be super clear